packages/deprecated-component-library-helpers/styles/layout.scss

```diff
@@ -1,4 +1,4 @@
-@import "type";
+@import "~@kaizen/deprecated-component-library-helpers/styles/type";
```

Comment:
This is a bit unrelated to the goal of the PR, but I noticed this was using a relative import but I think it's safer to explicitly do an absolute import (otherwise, it's a bit  non deterministic whether this file will come from deprecated-component-library-helpers or component-library). I want this to be consistent with the other imports in this package.

I had to look up the docs on this, but relative imports don't need the ".". It checks the same directory as the file first, and if it's not there it checks other root paths that have been loaded. It seems a lot less magical to me to just make it fully explicit where it's coming from.